src/User.ts

```diff
@@ -2,65 +2,13 @@
-    email_verified?: boolean;
-    /** End-User's gender. Values defined by this specification are female and male. Other values MAY be used when neither of the defined values are applicable. */
-    gender?: string;
-    /** End-User's birthday, represented as an ISO 8601:2004 [ISO8601‑2004] YYYY-MM-DD format. The year MAY be 0000, indicating that it is omitted. To represent only the year, YYYY format is allowed. Note that depending on the underlying platform's date related function, providing just year can result in varying month and day, so the implementers need to take this factor into account to correctly process the dates. */
-    birthdate?: string;
-    /** String from zoneinfo [zoneinfo] time zone database representing the End-User's time zone. For example, Europe/Paris or America/Los_Angeles. */
-    zoneinfo?: string;
-    /** End-User's locale, represented as a BCP47 [RFC5646] language tag. This is typically an ISO 639-1 Alpha-2 [ISO639‑1] language code in lowercase and an ISO 3166-1 Alpha-2 [ISO3166‑1] country code in uppercase, separated by a dash. For example, en-US or fr-CA. As a compatibility note, some implementations have used an underscore as the separator rather than a dash, for example, en_US; */
-    locale?: string;
-    /** End-User's preferred telephone number. E.164 [E.164] is RECOMMENDED as the format of this Claim, for example, +1 (425) 555-1212 or +56 (2) 687 2400. If the phone number contains an extension, it is RECOMMENDED that the extension be represented using the RFC 3966 [RFC3966] extension syntax, for example, +1 (604) 555-1234;ext=5678. */
-    phone_number?: string;
-    /** True if the End-User's phone number has been verified; otherwise false. When this Claim Value is true, this means that the OP took affirmative steps to ensure that this phone number was controlled by the End-User at the time the verification was performed. The means by which a phone number is verified is context-specific, and dependent upon the trust framework or contractual agreements within which the parties are operating. When true, the phone_number Claim MUST be in E.164 format and any extensions MUST be represented in RFC 3966 format. */
-    phone_number_verified?: boolean;
-    /** End-User's preferred postal address. The value of the address member is a JSON structure containing some or all of the members defined in the standard.
-     *
-     * @see https://openid.net/specs/openid-connect-core-1_0.html#AddressClaim
-     */
-    address?: Record<string, unknown>;
-    /** Time the End-User's information was last updated. Its value is a JSON number representing the number of seconds from 1970-01-01T0:0:0Z as measured in UTC until the date/time. */
-    updated_at?: number;
-
-    [claim: string]: unknown;
-}
+export type UserProfile = IdTokenClaims & Mandatory<OidcStandardClaims, "sub">;
```

Comment:
The OidcStandardClaims is redundant since IdTokenClaims already extends from it and makes `sub` required.

Additionally I'd prefer keeping this as an interface type even if it's empty

```suggestion
export interface UserProfile extends IdTokenClaims {}
````

Reply:
1. Agreed
2. This is going to make the linter complain : 
> An interface declaring no members is equivalent to its supertype
> eslint@typescript-eslint/no-empty-interface

Reply:
Oh interesting, I didn't realize `allowSingleExtends` was turned off here. The `export type` is ok.